requirements-optional.txt

```diff
@@ -31,6 +31,8 @@ fastapi>=0.75.0  # For web api
 uvicorn>=0.17.6  # For web api
 m2cgen>=0.9.0  # For model conversion
 evidently>=0.1.45.dev0  # for drift reporting
+comet_ml>=3.31.22 #comet ml tracking
+joblib >=1.2.0 #file pipeliing
```

Comment:
joblib exists in requirements.txt, i think you don't need insert insert in requirements-optional.txt

Reply:
deleted!